contracts/upgrades/stage/goerli/modules/SSVOperatorsV2.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.18;
+
+import "../../../../modules/SSVOperators.sol";
+import "../../../../libraries/Types.sol";
+
+contract SSVOperatorsV2 is SSVOperators {
+    using Types64 for uint64;
+    /****************************************/
+    /* Upgraded Operator External Functions */
+    /****************************************/
+
+    function getOperatorById(uint64 operatorId) external view returns (address, uint256, uint32, address, bool, bool) {
+        StorageData storage s = SSVStorage.load();
+        Operator memory operator = s.operators[operatorId];
+        address whitelisted = s.operatorsWhitelist[operatorId];
+        bool isPrivate = whitelisted == address(0) ? false : true;
```

Comment:
Just a solidity question (im curious not a CR) can you do this as well or its not valid in solidity:
bool isPrivate = !whitelisted == address(0)

Reply:
This file is just for testing in general right?

Reply:
Yes, only for testing

Reply:
The two lines of code you've provided are syntactically correct and will work as expected in Solidity